tests/testthat/test-derive_extreme_records.R

```diff
@@ -0,0 +1,34 @@
+# derive_extreme_records ----
+## derive_extreme_records Test 1: add last observation for each group ----
```

Comment:
Just wondering if there is enough test? Were you planning to add more? I see you start to 1? ;)

Reply:
I used "Test 1" due to our naming convention. I think the main functionality is covered by the unit test of `filter_extreme()` or did I miss something?